discord_bot/discord_bot.py

```diff
@@ -34,53 +36,94 @@ def load_config():
         with open(CONFIG_FILE_PATH) as f:
             return json.load(f)
 
-    def get_gateway(self):
+    def get_gateway(self, **kwargs):
```

Comment:
I really don't like this usage of `**kwargs`, it makes it very unclear what the contents will be since it can be anything.